app/services/create_message_service.rb

```diff
@@ -0,0 +1,24 @@
+class CreateMessageService
+  def initialize(params, current_user)
+    @current_user = current_user
+    @params = params
+  end
+
+  def create!
+    message_info = {
+      content: resource_params[:content],
+      user_id: @current_user.id
```

Comment:
As a comment, you can do `attr_reader :current_user`, so here you don't need to use class variables.

Reply:
Ok!